LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -213,13 +212,6 @@ private ILabAccount createTempAccountInternal(@NonNull final TempUserType tempUs
 
         final String password = getPassword(tempUser);
 
-        // Adding a wait to finish temp user creation
-        try {
-            Thread.sleep(TEMP_USER_CREATION_WAIT);
-        } catch (final InterruptedException e) {
-            e.printStackTrace();
-        }
-
```

Comment:
So this is no longer needed?

Reply:
There's a different wait already happening within the code, in the Lab account rule. This change essentially cuts the wait time in half